examples/emberjs/todomvc/config/environment.js

```diff
@@ -1,10 +1,10 @@
-/* jshint node: true */
+/* eslint-env node */
 
 module.exports = function (environment) {
 	var ENV = {
 		modulePrefix: 'todomvc',
 		environment: environment,
-		baseURL: null,
```

Comment:
`index.html` paths look correct with `rootURL`. I believe `baseURL` is deprecated 🤔